google-cloud-storage/lib/google/cloud/storage/project.rb

```diff
@@ -483,14 +485,26 @@ def create_bucket bucket_name, acl: nil, default_acl: nil,
         def signed_url bucket, path, method: nil, expires: nil,
                        content_type: nil, content_md5: nil, headers: nil,
```

Comment:
Should there be a note in the docs that `content_type` and `content_md5` are not used in V4?